identity-android/src/main/java/com/android/identity/SessionEncryption.java

```diff
@@ -51,50 +53,72 @@
 import co.nstant.in.cbor.model.ByteString;
 import co.nstant.in.cbor.model.DataItem;
 import co.nstant.in.cbor.model.Map;
-import co.nstant.in.cbor.model.Number;
 import co.nstant.in.cbor.model.UnicodeString;
 
 
 /**
  * A helper class for encrypting and decrypting messages exchanged with a remote
  * mDL prover, conforming to ISO 18013-5 9.1.1 Session encryption.
  */
```

Comment:
I would change this to say "Helper class for implementing session encryption according to ISO/IEC 18013-5:2021 section 9.1.1 Session encryption."